.gitattributes

```diff
@@ -0,0 +1,19 @@
+.git*           eol=lf export-ignore
+
+*.patch         eol=lf
```

Comment:
Wouldn't `*   eol=lf` be enough instead of listing each file type? I mean, there are no binary files anyway in this repository...